packages/zcli-themes/src/lib/createThemeImportJob.ts

```diff
@@ -28,11 +27,7 @@ export default async function createThemeImportJob (brandId: string): Promise<Pe
     return job
   } catch (e) {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const errors = (e as any).response.data.errors
-    for (const { code, title } of errors) {
-      error(`${chalk.bold(code)} - ${title}`)
-    }
-
-    error(new CLIError(e as axios.AxiosError))
+    const [{ code, title }] = (e as any).response.data.errors
+    error(`${chalk.bold(code)} - ${title}`)
```

Comment:
Curious, why this change?

Reply:
Once we call `error` it will error out, and to avoid that we need extra logic.
Even though we get an array of errors, it seems to only ever include one, so to simplify the code I'm just picking the first one.